backend/server.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const dotenv = require('dotenv');
+const cors = require('cors');
+// const proxy = require('http-proxy-middleware');
```

Comment:
please try not to leave commented code in PR without notes with reason why it should be left

Reply:
done